utils/api/benefitHandler.ts

```diff
@@ -750,14 +750,14 @@ export class BenefitHandler {
               allResults.client.gis.entitlement.type =
                 EntitlementResultType.FULL
 
-              allResults.partner.alw.cardDetail.collapsedText.push(
-                this.translations.detailWithHeading
-                  .calculatedBasedOnIndividualIncome
-              )
-              allResults.partner.alw.cardDetail = partnerAlw.cardDetail
-              allResults.partner.alw.entitlement.result = partnerAlwCalcSingle
-
               if (partnerAlwCalcSingle > 0) {
+                allResults.partner.alw.cardDetail.collapsedText.push(
+                  this.translations.detailWithHeading
+                    .calculatedBasedOnIndividualIncome
+                )
+                allResults.partner.alw.cardDetail = partnerAlw.cardDetail
```

Comment:
I am thinking these two lines should be outside. check with Vero or Lorelei because the summary shows the partner is eligible for alw but the card says isn't. 
the allResults shows partner card eligible but income too high

Reply:
That makes sense. I put the two lines outside the if statement.